test/test_wiki_trac_rst_convert.py

```diff
@@ -107,6 +107,168 @@ def test_several_trac_wiki_rst_links_with_content(self):
+            '`<Administrative>`_\n',
+            '[wiki:Administrative Administrative]'
+        )
+        self.assertConvertedContent(
+            '`<Administrative>`_\n',
+            '[wiki:"Administrative"]'
+        )
+        self.assertConvertedContent(
+            '`<Administrative-AllHandMeeting-Past>`_\n',
+            '[wiki:"Administrative/AllHandMeeting/Past"]'
+        )
+        self.assertConvertedContent(
+            '`<Infrastructure-Services-FileServer>`_\n',
+            '`[wiki:Infrastructure/Services/FileServer]`:trac:'
+        )
+        self.assertConvertedContent(
+            '`Overton <Infrastructure-Machines-Overton>`_\n',
+            '`[wiki:Infrastructure/Machines/Overton Overton]`:trac:'
+        )
+
+    def test_trac_ticket(self):
+        """
+        Trac ticket references are converted to a hyperlink.
+        This use case requires `config.py` with the following setting:
+
+        TRAC_TICKET_PREFIX = 'https://trac.chevah.com/ticket/'
+        """
+        self.assertConvertedContent(
+            '`Trac #738 <https://trac.chevah.com/ticket/738>`_\n',
+            ':trac:`#738`'
+        )
+
+    def test_heading(self):
+        """
+        Converts headings to RST, which have an equal-sign-underline.
+        Also handles the multiline case.
+
+        Headings in TracWiki have single equal signs around them.
+        """
+        self.assertConvertedContent(
+            'Heading\n=======\n',
+            '= Heading ='
+        )
+        self.assertConvertedContent(
+            'Policy and Process\n'
+            '==================\n\n'
+            'Some text\n',
+
+            '= Policy and Process =\n\n'
+            'Some text'
+        )
+
+    def test_subheading(self):
+        """
+        Converts subheadings to RST, which have a dash-underline.
+        Also handle the multiline case.
+
+        Subheadings in TracWiki have double equal signs around them.
+        """
+        self.assertConvertedContent(
+            'Subheading\n----------\n',
+            '== Subheading =='
+        )
+        self.assertConvertedContent(
+            'Subheading\n----------\n\n'
+            'Some text\n',
+
+            '== Subheading ==\n\n'
+            'Some text'
+        )
+
+    def test_list_indented(self):
+        """
+        Un-indents list items that are indented by one space exactly.
+
+        This is to avoid RST interpreting lists indented by one space
+        as quotations; we want them as unquoted lists instead.
+        """
+        self.assertConvertedContent(
+            "* item 1\n"
+            "* item 2\n"
+            "* item 3\n",
+
+            " * item 1\n"
+            " * item 2\n"
+            " * item 3\n"
+        )
+
+    def test_list_after_paragraph(self):
+        """
+        Separates lists from paragraphs by one empty line.
+
+        In RST, when lists follow a paragraph without an empty line
+        inbetween, they fail to parse as lists.
```

Comment:
I will leave it as is (add an empty line between), because GitHub does some very strange things with the syntax. 

It either:
* (not pictured) does not parse it as a list, if no newline nor one-space indent before list items, or
* it makes the preceding sentence bold and italic:

![gh-rst](https://user-images.githubusercontent.com/74835/106118987-5ae8ef80-615d-11eb-86e9-47f85d469841.png)